suzieq/poller/worker/services/arpnd.py

```diff
@@ -10,6 +10,16 @@
 class ArpndService(Service):
     """arpnd service. Different class because minor munging of output"""
 
+    def _clean_aos_data(self, processed_data, _):
+
+        entry, = processed_data
+
+        entry['ipAddress'] = entry.get('ip_address', '-1')
+        entry['macaddr'] = entry.get('hardware_addr', '-1')
+        entry['oif'] = entry.get('port', '-1') # or 'port' for physical interface
+
+        return processed_data
```

Comment:
Do you think it could be possible to retrieve also the `state` field (reachable, permanent .. )?

Reply:
Added ´state´ column from ´type´ field:
STATIC = permanent
DYNAMIC = reachable